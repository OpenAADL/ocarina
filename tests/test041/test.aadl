package p1 
public
   data d1;
   --  Ocarina should not hang indefinitely when parsing the incomplete
   --  component above

   data implementation d1.impl1;
   --  Idem...

end p1;
