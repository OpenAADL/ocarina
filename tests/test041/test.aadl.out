test.aadl:3:11: parsing Component_Type, unexpected token ';'
test.aadl:7:32: parsing Component_Implementation, unexpected token ';'
test.aadl:10:07: parsing Package_Specification, Empty packages are not allowed
Cannot parse AADL specifications
